File: SobrecargaQuadrilatero/src/Quadrilatero.java
public class Quadrilatero {
    public static void area(double lado){
        System.out.println("Area do quadrado: " + Math.pow(lado, 2));
    }

    public static void area(double lado1, double lado2){
        System.out.println("Area do quadrado: " + (lado1 + lado2));
    }

    public static void area(double baseMaior, double baseMenor, double altura){
        System.out.println("Area do trapézio: " +(((baseMaior + baseMenor)*altura) / 2));
    }

}

File: Quadrilatero/Quadrilatero.cs
namespace Quadrilatero
{
    public class Quadrilatero
    {
        public static void Area(double lado)
        {
            Console.WriteLine("Area do quadrado: {0}", Math.Pow(lado,2));
        }

        public static void Area(double lado1, double lado2)
        {
            Console.WriteLine("Area do quadrado: {0}", lado1 + lado2);
        }

        public static void Area(double baseMenor, double baseMaior, double altura)
        {
            Console.WriteLine("Area do trapézio: {0}",((baseMaior + baseMenor) * altura) / 2);
        }
    }
}
